src/view/lesson_overview.js

```diff
@@ -7,51 +7,6 @@ var T = require("../util/translate").T;
 var ProgressManager = require("../backend/progress_manager");
 
 var LessonOverview = React.createClass({
-  _styles: {
```

Comment:
👍 for getting rid of these inline styles. Eventually we should get rid of all of them, I think.